src/main/java/com/yahoo/bullet/record/BulletRecord.java

```diff
@@ -542,11 +542,11 @@ public BulletRecord setListOfDoubleMap(String field, List<Map<String, Double>> o
      * @return This object for chaining.
      */
     public BulletRecord set(String field, BulletRecord that, String thatField) {
-        return set(field, that.get(thatField));
+        return forceSet(field, that.get(thatField));
```

Comment:
You can still use set?

Reply:
I am using the convention  that if I ever do an unsafe set, I use forceSet. Since we allow forceSet now, the ```that`` BulletRecord can have unsafe objects in it. So using forceSet to copy it.